bootstrap/spigot/src/main/java/org/geysermc/geyser/platform/spigot/GeyserSpigotPlugin.java

```diff
@@ -79,6 +79,7 @@
 import java.util.UUID;
 import java.util.logging.Level;
 
+@SuppressWarnings("NullableProblems") // getResource() from JavaPlugin can return null
 public class GeyserSpigotPlugin extends JavaPlugin implements GeyserBootstrap {
```

Comment:
Does this need to be applied class-wide?

Reply:
https://github.com/GeyserMC/Geyser/pull/4238/commits/27c9902b082fd4c49228d1769dc15d8312bce353 fixed properly here, removed the suppression